sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -932,11 +932,20 @@ object SQLConf {
         "a positive value, a running query will be cancelled automatically when the timeout is " +
         "exceeded, otherwise the query continues to run till completion. If timeout values are " +
         "set for each statement via `java.sql.Statement.setQueryTimeout` and they are smaller " +
-        "than this configuration value, they take precedence.")
+        "than this configuration value, they take precedence. After set this config, you may be " +
+        "interesting in spark.sql.thriftServer.interruptOnCancel which can help interrupt task.")
```

Comment:
Can you use `THRIFTSERVER_FORCE_CANCEL.key` with string interpolation? You can place `THRIFTSERVER_FORCE_CANCEL` before `THRIFTSERVER_QUERY_TIMEOUT` if forward referencing becomes a problem here.